vpc.tf

```diff
@@ -0,0 +1,17 @@
+module "vpc" {
+    source = "github.com/terraform-aws-modules/terraform-aws-vpc"
```

Comment:
remove github.com (since it's auto expanding github)

Reply:
Done